cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -54,9 +54,9 @@ func loadConfig(data []byte) (*config.KubeSchedulerConfiguration, error) {
 		cfgObj.TypeMeta.APIVersion = gvk.GroupVersion().String()
 		switch cfgObj.TypeMeta.APIVersion {
 		case configv1beta2.SchemeGroupVersion.String():
-			klog.InfoS("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.28")
+			logger.Info("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.28")
 		case configv1beta3.SchemeGroupVersion.String():
-			klog.InfoS("KubeSchedulerConfiguration v1beta3 is deprecated in v1.26, will be removed in v1.29")
+			logger.Info("KubeSchedulerConfiguration v1beta3 is deprecated in v1.26, will be removed in v1.29")
```

Comment:
Nit:
Maybe we should be using the same message:

```
"Using deprecated KubeSchedulerConfiguration version", "version", "v1beta2", "toBeRemovedInk8sVersion", "v1.28"
```

Although I'm not convinced.

Reply:
I am not sure :thinking:  I can do the change if you prefer

Reply:
Not worth, we will removing these lines in the next couple of releases.